gum/gumtls-priv.h

```diff
@@ -0,0 +1,18 @@
+/*
+* Copyright (C) 2010 Eloi Vanderbeken <eloi.vanderbeken@synacktiv.com>
+*
+* Licence: wxWindows Library Licence, Version 3.1
+*/
+
+#ifndef __GUM_TLS_PRIV_H__
+#define __GUM_TLS_PRIV_H__
+
+#include <gum/gumdefs.h>
+
+G_BEGIN_DECLS
+
+void _gum_tls_init (void);
```

Comment:
This should be declared `G_GNUC_INTERNAL` (look for it in other files for an example).